reference/src/client/test/sa_engine_pkey_decrypt.cpp

```diff
@@ -91,6 +91,9 @@ TEST_P(SaEnginePkeyEncryptTest, encryptTest) {
     std::shared_ptr<EVP_MD_CTX> evp_md_verify_ctx(EVP_MD_CTX_new(), EVP_MD_CTX_free);
     int result = EVP_PKEY_encrypt(encrypt_pkey_ctx.get(), encrypted_data.data(), &encrypted_data_length, data.data(),
             data.size());
+    if (result == -2)
```

Comment:
Recommend using defined enums instead of magic numbers for code readability.

Reply:
Done